app/src/main/java/com/SecUpwN/AIMSICD/AIMSICD.java

```diff
@@ -93,7 +93,6 @@
     private ActionBarDrawerToggle mDrawerToggle;
     private CharSequence mDrawerTitle;
     private CharSequence mTitle;
-    public static ProgressBar mProgressBar;
```

Comment:
>@SecUpwN This one here is against any standards of object orientated design

Question is, is OO bad for this purpose or is this purpose bad for OO :> I'm totally for this code to stay. It's clear and difficult to break even if violates basic OO principles.